lib/assets/javascripts/twine.coffee

```diff
@@ -147,6 +166,13 @@
         return context
       node = node.parentNode if child
 
+  getIndexesForElement = (node) ->
+    firstContext = null
+    while node
+      return elements[id]?.indexes if id = node.bindingId
```

Comment:
Since we assign the same indexes to all child nodes that have any bindings, if a node has a bindingId it will also have the set of indexes to use, if any.